mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -200,6 +200,7 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_6085 setLivingFlag (IZ)V
 		ARG 1 mask
 		ARG 2 value
+	METHOD method_6086 canApplyPotion ()Z
```

Comment:
I'd lile to use the word `immune` somewhere in there. Anyone got a suggestion?

Reply:
`isPotionImmune` ?

Reply:
It's reversed, entities that return false are immune